qiime2/sdk/typemap.py

```diff
@@ -0,0 +1,48 @@
+import random
+from itertools import combinations
+import networkx as nx
```

Comment:
Oops, sorry! When I was looking at the failing travis log earlier I saw a message about a missing networkx dependency. Looks like you aren't actually using networkx here, just left the import in. Can you please remove this?